apps/s_server.c

```diff
@@ -2767,7 +2767,7 @@ static int sv_body(int s, int stype, int prot, unsigned char *context)
 #ifdef CHARSET_EBCDIC
             ebcdic2ascii(buf, buf, i);
 #endif
-            l = k = 0;
+            l = 0;
```

Comment:
Again, the assignment to `k` has the smell of initialisation. But here it would be nicer to declare `int k` in the `for (;;)` loop, and then we won't have to worry about what happens if we refactor later